### ISSUE TITLE
some script command error in readme  and question about wer number

### ISSUE THREAD

**Issue by Reikin** [sandy1990418/ChineseTaiwaneseWhisper#7]
I'm really happy to see a project related to Whisper that supports mixed Mandarin and Taiwanese Hokkien.

I have a few questions I'd like to ask.
I followed the README to train on Common Voice 13, but the resulting WER is between 100 and 200. Is this normal?
How can I effectively reduce it?

Additionally, in the README, the parameters --dataset_name and --youtube_data_dir seem to no longer exist.

**Comment by sandy1990418** [comment]
@Reikin 

I don’t think this is normal. I’m not sure if recursion happens at the end. You might want to check whether add_special_tokens=True, verify your dataset quality (e.g., proper alignment, removing noisy data, and applying VAD), or review your LoRA setup and corresponding hyperparameter settings (e.g., rank size, learning rate, gradient clipping). If the issue persists, you may also want to check your decoding settings (like max_length and repetition_penalty) to prevent unwanted looping in inference.

And thank you so much for catching that! I really appreciate it. I’ll take some time to fix it and update the correct parameters. 😊

**Comment by Reikin** [comment]
@sandy1990418 
Hello, may I ask what WER you obtained when running the command?

I used the following command from the README:
```
python scripts/train.py --model_name_or_path "openai/whisper-small" \
                        --language "chinese" \
                        --dataset "common_voice_13_train" \
                        --dataset_dir "./data" \
                        --output_dir "./whisper-finetuned-zh-tw" \
                        --num_train_epochs 3 \
                        --per_device_train_batch_size 16 \
                        --learning_rate 1e-5 \
                        --fp16 \
                        --timestamp False
```
Running this command, my initial WER was 110. The dataset I used was from Common Voice 13, without any custom datasets.

I'm curious about the WER you obtained using this command. Also, do you have any better WER results? If so, what command did you use?

My GPU is NVIDIA RTX 4060 Laptop、i9-13900H

Thank you very much!

**Comment by sandy1990418** [comment]
@Reikin 

Sorry, I don't have the record for whisper-small at the moment, but I do have the results for whisper-large-v3-turbo. You can refer to this [Whisper model](https://huggingface.co/sandy1990418/whisper-large-v3-turbo-chinese) on HuggingFace to check the results.

The device used was T4 16GB VRAM

BTW, `per_device_train_batch_size ` I set to 8, but I didn't adjust any other parameters.
